### ISSUE TITLE
Document an example demonstrating integration with twirl

### ISSUE THREAD

**Issue by girdharshubham** [zio/zio-http#423]
# Description
Twirl is a powerful Scala-based template engine. We'd need a document and an example demonstrating integration of zio-http with twirl.

# Example
* A sample application
```scala
package edu.self.training

import play.twirl.api._
import zhttp.http._
import zhttp.service.Server
import zio.{ExitCode, URIO}

object ZIOHttp extends zio.App {
  val app: HttpApp[Any, Nothing] = HttpApp.collect {
    case Method.GET -> Root => {
      val response: UResponse = Response
        .text(html
          .index("John Doe", 42)
          .toString()
        )

      response
        .asInstanceOf[UHttpResponse]
        .copy(headers = List(Header.custom("content-type", "text/html")))
    }
  }

  override def run(args: List[String]): URIO[zio.ZEnv, ExitCode] =
    Server.start(80, app).exitCode

}
```
* Build configuration
```scala
project.enablePlugins(SbtTwirl)
```
* Plugin
```scala
addSbtPlugin("com.typesafe.sbt" % "sbt-twirl" % "1.5.1")
```

# Specs
* The example should be complex enough to demonstrate capabilities provided by Twirl and a ZIO-backed implementation of the templating process

**Comment by brbrown25** [comment]
If I have some time I’d be interested in picking this up :)

**Comment by domdorn** [comment]
@brbrown25 are you still on to this?

**Comment by brbrown25** [comment]
> @brbrown25 are you still on to this?

I didn't get a chance to, so feel free to pick it up. Otherwise I'll have some time this week to hack on it.

**Comment by brbrown25** [comment]
@tusharmath I'm gonna start working on this today and hope to have a PR by EOW, depending on schedule. I'm think this is a change that would need to be added to the example project?

**Comment by amitksingh1490** [comment]
Available in 1.0.0.0-RC18
